codegen/jsonprotocol.py

```diff
@@ -20,6 +34,9 @@ def rust_struct(self, name, shape):
                 rust_type =  member['shape']
                 if not JsonProtocolParser.is_required(shape, mname):
                     rust_type = "Option<" + rust_type + ">"
+                if mname in KEYWORDS:
+                    self.append("\t#[serde(rename=\""+ mname+"\")]")
```

Comment:
just wanted to make sure it was clear what was going on here and to make sure my approach seemed ok. This prevents compiler errors when a json member's name is a reserved keyword. I'm renaming it to `_thename` and having serde map it correctly with the rename annotation.